jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -141,7 +151,7 @@ def user_api(self):
         if self.dev_bg_support:
             expose += ['background', 'bg_wavelength_dependent']
         if self.dev_subpixel_support:
-            expose += ['subpixel']
+            expose += ['aperture_method_items', 'aperture_method_selected']
```

Comment:
`aperture_method` should be created in init as a `SelectPluginComponent` and then listed here as well as the API docstring (but the traitlets themselves should not be).